### ISSUE TITLE
Option to keep new line between selectors in SASS

### ISSUE THREAD

**Issue by MiincGu** [austincheney/prettydiff#43]
Hi,

I just want to know if there is an option to keep new line between selectors in sass/css/less files.
Like below:

```
.a {
  color: #fff;

  .b {
    color: #999;
  }
}
```

not

```
.a {
  color: #fff;
  .b {
    color: #999;
  }
}
```

cc @Glavin001

**Comment by austincheney** [comment]
This feature does not yet exist, but I can work this. This should be a standard feature and not something hidden behind an option.

**Comment by austincheney** [comment]
Would you mind looking at this QA version and letting me know if you encounter any defects?

http://prettydiff.com/ignore/testlocation/prettydiff.com.xhtml

**Comment by MiincGu** [comment]
Chrome, Mac OS X

When choose beatuify, and input this:
```
.a {
    background: gray;
    
    .b {
        background: white;
    }
}
```

then click excute, in console, it keeps telling me 
```
Uncaught ReferenceError: elseline is not defined
```

in dom.js:1146

**Comment by austincheney** [comment]
I am so sorry. I had some defects. Please try again:

http://prettydiff.com/ignore/testlocation/prettydiff.com.xhtml

**Comment by MiincGu** [comment]
It's ok now. Thanks.

**Comment by austincheney** [comment]
Cool. I have some clean up to do in the code and I will launch this tomorrow.

**Comment by austincheney** [comment]
Change published as npm v1.7.2
